### ISSUE TITLE
strange null Warnings 

### ISSUE THREAD

**Issue by JJ-Author** [imas/rdflint#112]
seems not to work an returns strange warnings
https://github.com/dbpedia/format-mappings/runs/477652228?check_suite_focus=true

```
Run rdflint3s
##[error]Process completed with exit code 1.
Run rdflint
tarql/1.ttl
  ERROR  RDF parse warning: [line: 1, col: 1] null

tarql/2.ttl
  ERROR  RDF parse warning: [line: 1, col: 1] null

tarql/3.ttl
  ERROR  RDF parse warning: [line: 1, col: 1] null

##[error]Process completed with exit code 1.
```

**Comment by foooomio** [comment]
Based on my research, the warnings appear when neither the command line option `baseuri` nor the config option `baseUri` is specified.

Is it intentional? @takemikami

**Comment by JJ-Author** [comment]
Thanks for the quick reply. I don't fully understand this. When reading from a file usually the base URI is set to the filename / filepath. It could additionally make sense to have a config file in the action which works out of the box. For validation / linting the baseUri is not important. I fail to understand the config file since the docu is in Japanese and Google refuses to translate it ^^. So yes the issue might be in a right place now, but I think a working default config in the action setup makes sense, doesn't it?

**Comment by takemikami** [comment]
This strange warnings is a bug, and I will fix correctly in default configuration.

If you need to use rdflint shortly,
please put configuration file like following, and you need avoid this bug.

rdflint-config.yml
```
baseUri: http://example.com/
```

**Comment by takemikami** [comment]
fixed, at v0.1.4
